### ISSUE TITLE
Accessibility - Contrast in Workflow Delete and Undo buttons

### ISSUE THREAD

**Issue by csmith-he** [archesproject/arches#9010]
NB Although this relates to workflows which are held in arches-her repo, the styling changes will need to be made in accessibility.css which is held in the arches repo.

Issue found in instance of Arches 6.1.x and will be fixed as part of accessibility work

**Steps to Reproduce:**

1. Navigate to any Workflow
2. Using keyboard, tab through page contents to the 'Delete' button (which deletes the entire workflow)
3. Observe the colour contrast which indicates keyboard focus on this component (Undo button visibility is identical to this)

**Expected Result:**
Keyboard focus should be made visible by highly contrasting colour changes or thick, colour-contrasting borders around the component receiving focus. 

**Actual Result:**
The component changes to a red too similar to the red of the text when receiving focus, which could obscure the text in the component for some visually impaired users. 

**Evidence:**

![image](https://user-images.githubusercontent.com/38533555/194288158-417c49ed-b0e5-4afa-994f-66f0c7b84226.png)
